scandium-core/src/main/java/org/eclipse/californium/scandium/dtls/cipher/CipherSuite.java

```diff
@@ -471,8 +472,12 @@ public static int getOverallMaxCiphertextExpansion() {
 	/**
 	 * Get a list of all supported PSK cipher suites.
 	 * 
-	 * @param recommendedCipherSuitesOnly {@code true} use only recommended cipher suites
-	 * @return list of all supported PSK cipher suites. Ordered by their definition above.
+	 * @param recommendedCipherSuitesOnly {@code true} use only recommended
+	 *            cipher suites
+	 * @return list of all supported PSK cipher suites. Ordered by their
+	 *         definition above.
+	 * @deprecated use
+	 *             {@link #getCipherSuitesByKeyExchangeAlgorithm(boolean, KeyExchangeAlgorithm...)}
 	 */
 	public static List<CipherSuite> getPskCipherSuites(boolean recommendedCipherSuitesOnly, boolean ecdhePsk) {
```

Comment:
should we add the @deprecated annotation ?

Reply:
OK